src/java.base/share/classes/javax/crypto/CipherSpi.java

```diff
@@ -224,18 +224,18 @@ public abstract class CipherSpi {
     public CipherSpi() {}
 
     /**
-     * Sets the mode of this cipher.
+     * Sets the mode of this {@code Cipher}.
```

Comment:
Same comment as in the one in Cipher.java, prefer to leave cipher alone.

Reply:
I believe we're doing `{@code Cipher}` object.